includes/class-wc-payments-onboarding-service.php

```diff
@@ -277,6 +277,13 @@ public function create_embedded_kyc_session( array $self_assessment_data, bool $
 		);
 		$actioned_notes = self::get_actioned_notes();
 
+		$capabilities = $this->get_capabilities_from_request();
```

Comment:
Why do we need to do it here also when we do it at the `finalize_embedded_connection` step?

Reply:
Since the capabilities are not passed to the `/kyc/finalize` REST route, I believe it would be better to handle the payment methods update before account creation and remove it from `finalize_embedded_connection`. Updated.

Reply:
Ok. Either one or the other.

Reply:
Updated!